design-system/react/src/utils/css.ts

```diff
@@ -0,0 +1,7 @@
+export function getCSSValue(value?: string | number | undefined) {
```

Comment:
I would change the function name to `toCSSValue` , which imples a conversion/parse style...

`getCSSValue` sounds like a function that would take an input element, and give me computed CSS value from it

if you think it makes sense lets change